pyproject.toml

```diff
@@ -117,6 +117,7 @@ extra-dependencies = [
   "langdetect",  # TextLanguageRouter and DocumentLanguageClassifier
   "sentence-transformers>=2.2.0",  # SentenceTransformersTextEmbedder and SentenceTransformersDocumentEmbedder
   "openai-whisper>=20231106",  # LocalWhisperTranscriber
+  "python-pptx",  # PPTXToDocument
```

Comment:
This is not required if you're using `LazyImport`. :)

Reply:
ahh right ... my bad 😅 I'll go ahead and remove!

Reply:
removed!

Reply:
Hey @silvanocerza I think I need this under the extra deps to be able to run the linting and tests. Is there a better place for me to add the dep to get the tests to run?

Reply:
Oh wait! I just noticed that this was under the extra dependencies. 🤦 

My bad, this is the correct place for this. I thought it was the default dependencies.
You actually need to do this if you're using `LazyImport` 😓

Reply:
Adding it back! :D